ingest-app/src/cmr/ingest/api/translation.clj

```diff
@@ -1,15 +1,18 @@
 (ns cmr.ingest.api.translation
   "Defines an API for translating metadata between formats."
   (:require
-    [clojure.test.check.generators :as test-check-gen]
-    [cmr.common.mime-types :as mt]
-    [cmr.common.services.errors :as errors]
-    [cmr.umm-spec.legacy :as umm-legacy]
-    [cmr.umm-spec.test.umm-generators :as umm-generators]
-    [cmr.umm-spec.umm-spec-core :as umm-spec]
-    [cmr.umm-spec.util :as u]
-    [cmr.umm-spec.versioning :as ver]
-    [compojure.core :refer :all]))
+   [clojure.java.io :as io]
+   [clojure.test.check.generators :as test-check-gen]
+   [cmr.common.cache :as cache]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.services.errors :as errors]
+   [cmr.common.xml.xslt :as xslt]
+   [cmr.umm-spec.legacy :as umm-legacy]
+   [cmr.umm-spec.test.umm-generators :as umm-generators]
+   [cmr.umm-spec.umm-spec-core :as umm-spec]
+   [cmr.umm-spec.util :as util]
+   [cmr.umm-spec.versioning :as versioning]
+   [compojure.core :refer :all]))
 
 (def concept-type->supported-formats
   "A map of concept type to the list of formats that are supported both for input and output of
```

Comment:
This doesn't match the documentation that says ISO-19115 is supported as an output format, but not an input format.

Reply:
Good point. Fixed the validation to not allow ISO19115 as input format for granule translation.